python-spec/src/somacore/experiment.py

```diff
@@ -66,6 +66,17 @@ class Experiment(
     spatial = _mixin.item[_SceneColl]()  # TODO: Discuss the name of this element.
     """A collection of named spatial scenes."""
 
+    obs_spatial_presence = _mixin.item[_DF]()
+    """A dataframe that stores the presence of obs in the spatial scenes.
+
+    This provides a join table for the obs ``soma_joinid`` and the scene names used in
+    the ``spatial`` collection. This dataframe must contain dimensions ``soma_joinid``
```

Comment:
the use of the word `dimension` is confusing as used here.  SOMA does not have the concept of a dimension as you mean it (ie, an index), outside of joinids.

Two suggestions:
1. The abstract spec needs to be updated to clarify the new concepts and changes implied by SpatialDataFrame, et al.
2. We should nail down our terminology and use it consistent.  The abstract spec mentions "index" and "indexing", but does not use the TileDB-specific "dimension" terminology.

Reply:
+1